packages/salesforcedx-vscode-core/README.md

```diff
@@ -73,6 +74,13 @@ To edit your workspace settings, select **Code** > **Preferences** > **Settings*
 
 To stop Salesforce CLI success messages from showing as pop-up information messages, click **Show Only in Status Bar** in a success message. This button overrides the `salesforcedx-vscode-core.show-cli-success-msg` value in your Default Settings. It changes the Workspace Settings value to `false`. Setting this value to `false` makes the success messages appear in the status bar (in VS Code’s footer) instead of as information messages. If you decide that you liked the information messages after all, change the value back to `true`.   
 
+## Activate Demo Mode
+If you’re setting up a machine that will be used to give demos at a conference (or otherwise used by people you don’t know), set up demo mode. When in demo mode, VS Code warns users who authorize business or production orgs of the potential security risks of using these orgs on shared machines.  
+
+To activate demo mode, add an environment variable called `DEMO` and set its value to `true`.  
```

Comment:
To be more precise, the environment variable needs to be `SFDX_ENV=DEMO`. See https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-core/src/modes/demo-mode.ts#L9 for more information.

I also don't know if we need to mention this (or link to the release notes for the SFDX CLI) but setting that environment variable also has other repercussions on the behavior of the SFDX CLI (not just VS Code extensions).

Reply:
Demo mode actually isn't in the Salesforce CLI docs. I considered linking to https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_cli_env_variables.htm?search_text=environment%20variables , but this environment variable isn't currently documented there. @ekapner , do you have plans to document `SFDX_ENV=DEMO` somewhere that I should link to?